src/middlewares/polaris-middleware-creator.ts

```diff
@@ -1,22 +1,21 @@
 import { GraphQLResolveInfo } from 'graphql';
-import { PolarisMiddleware } from './polaris-middleware';
+import { PolarisContext, PolarisMiddleware } from './polaris-middleware';
 
-export const createMiddleware = <TContext>(middleware: PolarisMiddleware<TContext>) => {
+export const createMiddleware = (middleware: PolarisMiddleware) => {
     return async (
         resolve: (
             root: any,
             args: { [argName: string]: any },
-            context: TContext,
+            context: PolarisContext,
```

Comment:
Can someone add properties to context? If so we want to give them the abillity to pass their context type
Can you leave the generic here and give PolarisContext as default type?